packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:html';
+
+class DragScroll {
+  var mousemove = 'mousemove';
+  var mouseup = 'mouseup';
+  var mousedown = 'mousedown';
+  var EventListener = 'EventListener';
+
+// ??? what are these ???
+//  var addEventListener = 'add'+EventListener;
+//  var removeEventListener = 'remove'+EventListener;
+
+
+
+  num newScrollX;
+  num newScrollY;
+
+  var dragged = [];
+
+  void reset(int i, /*type?*/ eventListener) {
+    while( i < dragged.length) {
+      eventListener = dragged[i++];
+      eventListener = eventListener.container ?? eventListener;
```

Comment:
what is .container? again knowing what the type of eventListener is would help (which is probably very misnamed at the moment)

Reply:
something crazy. I think you should be able to remove it. you might need to add it back some equivalent version. potentially the JS library was letting you add a container expando to elements to indicate the container associated with an element. *shrug*.